src/components/MoviesList/MoviesList.js

```diff
@@ -1,6 +1,20 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard/MovieCard';
 
-export const MoviesList = () => (
-  <>Put the list here</>
-);
+let arr = [];
+
+export const MoviesList = (props) => {
+  arr = props.items;
+
+  return (
+    <div className="movies">
+      {arr.map(item => <MovieCard key={item.imdbId} items={item} />)}
```

Comment:
```suggestion
      {items.map(item => <MovieCard key={item.imdbId} items={item} />)}
```

Reply:
```suggestion
      {arr.map(item => <MovieCard key={item.imdbId} item={item} />)}
```
Ты передаешь один элемент, одни элемент - item, а не items